hack/update-codegen.sh

```diff
@@ -61,14 +55,14 @@ chmod +x "${CODEGEN_PKG}"/generate-groups.sh
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
 "${CODEGEN_PKG}"/generate-groups.sh "deepcopy,client,informer,lister" \
   github.com/google/knative-gcp/pkg/client github.com/google/knative-gcp/pkg/apis \
-  "pubsub:v1alpha1 pubsub:v1beta1 messaging:v1alpha1 messaging:v1beta1 events:v1alpha1 events:v1beta1 broker:v1beta1 intevents:v1alpha1" \
+  "messaging:v1alpha1 messaging:v1beta1 events:v1alpha1 events:v1beta1 broker:v1beta1 intevents:v1alpha1" \
```

Comment:
intevents should also have v1beta1?

Reply:
Yes it should! But I think it should be in a distinct PR, not this one. I'll send that as soon as this one is in.

Reply:
sounds good!

Reply:
#1146